### DISCUSSION TITLE
Adventure map, System Options window, change to texts for instant combat settings buttons

### DISCUSSION THREAD

**Discussion by LeHerosInconnu** [ihhub/fheroes2#10276] [General]
### Preliminary checks

- [X] I've checked that there aren't [**other open issues**](https://github.com/ihhub/fheroes2/issues?q=is%3Aissue) on the same topic.

### Describe the problem requiring a solution

In fheroes2, the texts of the instant combat settings buttons are not clear.

### Describe the possible solution

First, I suggest changing the text “Battles” by the text “Combat” (without “s” at the end) for the three buttons.

For this one:

![Auto and instant combat modes 004](https://github.com/user-attachments/assets/08077123-8c39-4f3a-b834-3da9799d074a)

No change ("Manual" is okay).

For this other one:

![Auto and instant combat modes 003](https://github.com/user-attachments/assets/aa401fe7-482d-4bce-95ab-a9f7bd720ae5)

Change the text to:

"Instant,
No Spells"

Instead of:

"Auto, No
Spells"

For the last one:

![Auto and instant combat modes 005](https://github.com/user-attachments/assets/ccb58046-c5a5-4fae-979b-b820f7386364)

Change the text to:

"Instant,
With Spells"

Instead of:

"Auto Resolve"

**Comment by ihhub** [comment]
Hi @zenseii , could you please provide your opinion about this?

**Comment by zenseii** [comment]
@ihhub, this should be converted into a discussion due to no one on the team agreeing ho this change in previous discussions.
